sshnp_docker_demo/README.md

```diff
@@ -0,0 +1,262 @@
+# sshnp_docker_demo
+
+This is a demo of sshing into one docker container from another docker container without having ports open (not even port 22). There are three docker containers: `sshnp`, `sshnpd`, and `sshrvd`. This demo involves using [sshnp](https://github.com/atsign-foundation/sshnoports).
+
+## Getting Started
+
+Follow these steps to get sshnp working on your machine using Docker.
+
+### 1. Prerequisites
+
+1. Install [Docker](https://www.docker.com/products/docker-desktop/) and open it in the background. Check that you have docker installed. Run this in your terminal. Your output should be similar:
+
+```sh
+docker --version
+Docker version 23.0.5, build bc4487a
+```
+
+2. You will need 3 atSigns and their associated `.atKeys` files (one for each docker container). For each of your atSigns, put the `.atKeys` file into the `keys/` directory. For example, your file structure should be similar to:`sshnp/keys/@sshnp_key.atKeys`, `sshnpd/keys/@sshnpd_key.atKeys`, `sshrvd/keys/@sshrvd_key.atKeys`.
+
+### 2. Finding IP of sshrvd
+
+This is a tedious but short step where we need to do to find the IP of the sshrvd docker container.
+
+Let's build and run the `sshrvd` docker container by running
+
+```sh
+cd sshrvd
+sh db.sh
```

Comment:
I get that it's called db for Docker Build, but usually that would be DataBase. Maybe use a more explicit name.

Also why `sh db.sh` rather than `./db.sh`?

Reply:
Now uses `./` to execute scripts instead of `sh`
https://github.com/atsign-foundation/at_demos/pull/97/commits/1aaad8c04a64f11085e9a2619e7c1883c5789d4d

Reply:
All the scripts need to be `chmod +x` so that they're runnable

Reply:
@cpswan should the script also chmod +x then?

Reply:
You should be doing `chmod +x` *to* the scripts (so they're executable when cloned from git) not *from* the scripts.

Reply:
done https://github.com/atsign-foundation/at_demos/pull/97/commits/a7d0b6f82621935aa5d41f6fda37e8e907e92b73

and here too (i missed a couple) https://github.com/atsign-foundation/at_demos/pull/97/commits/ae38e9cf78d250d7ae6204d7a36e41c356228bd2